apps/wing-console/console/app/demo/index.w

```diff
@@ -0,0 +1,117 @@
+bring cloud;
+// bring redis;
```

Comment:
why aren't we testing redis?

Reply:
This is the same demo file we used in the previous repository. I guess we commented out redis because it needs docker and may be annoying.